packages/analytics/e2e/analytics.e2e.js

```diff
@@ -56,16 +56,6 @@ describe('analytics()', () => {
     });
   });
 
-  describe('setCurrentScreen()', () => {
```

Comment:
Can we keep this test in, just to make sure the old method is called during tests and nothing blows up